qpc/utils.py

```diff
@@ -340,18 +340,14 @@ def setup_logging(verbosity):
     else:
         log_level = logging.DEBUG
 
+    log_fmt = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
```

Comment:
I know that the log changes are mostly for the end user, but do you think we could benefit from adding "%(funcName)s:%(lineno)d" to formatter whenever we are on debug verbosity level?

Reply:
makes sense

Reply:
any thoughts on this https://github.com/quipucords/qpc/pull/195/commits/16f36451f4cdfa1f2e34f477e11b314b5d29edcd?